test/_data/close_sinks/src/a.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Interesting.  I'm guessing we never figured out why we couldn't have a rule test like the others?